SCCM-TSEnvironment.psm1

```diff
@@ -427,6 +428,10 @@ function Show-TSErrorDialog()
     Sets an Error and forces a reboot
     Show-TSErrorDialog -OrganizationName "My Organization" -CustomTitle "An Error occured during the things" -ErrorMessage "He's dead Jim!" -ErrorCode 123456 -TimeoutInSeconds 90 -ForceReboot $true
 
+    .EXAMPLE
+    Adds TSStepName which is required for SCCM 1901 TP and newer
+    Show-TSErrorDialog -OrganizationName "My Organization" -CustomTitle "An Error occured during the things" -ErrorMessage "That thing you tried...it didnt work" -ErrorCode 123456 -TimeoutInSeconds 90 -ForceReboot $false -TSStepName "My Step Name"
```

Comment:
Think is beneficial to recommend `_SMSTSCurrentActionName` as a good default parameter?

Reply:
Yes. I think that makes sense.